File: Qubit Server/dev/src/org/labrad/qubits/channeldata/IqData.java
package org.labrad.qubits.channeldata;

import org.labrad.qubits.channels.IqChannel;


public interface IqData extends Deconvolvable {
  public void setChannel(IqChannel channel);
  public int[] getDeconvolvedI();
  public int[] getDeconvolvedQ();
  public void checkLength(int expected);
}

File: QubitServer/src/main/scala/org/labrad/qubits/channeldata/IqData.scala
package org.labrad.qubits.channeldata

import org.labrad.qubits.channels.IqChannel

trait IqData extends Deconvolvable {
  def setChannel(channel: IqChannel): Unit
  def getDeconvolvedI(): Array[Int]
  def getDeconvolvedQ(): Array[Int]
  def checkLength(expected: Int): Unit
}
